antarest/core/config.py

```diff
@@ -417,38 +435,36 @@ class LauncherConfig:
     """
 
     default: str = "local"
-    local: Optional[LocalConfig] = None
-    slurm: Optional[SlurmConfig] = None
+    configs: Optional[List[LocalConfig | SlurmConfig]] = None
     batch_size: int = 9999
 
     @classmethod
     def from_dict(cls, data: JSON) -> "LauncherConfig":
+        launchers: List[LocalConfig | SlurmConfig] = []
         defaults = cls()
         default = data.get("default", cls.default)
```

Comment:
Now we need to check that default is actually defined in the list of configurations